### PR TITLE
Update tester script

### PR DESCRIPTION
**Pull request by mohawk2**
Mainly adds `openapi-client` which fails with current version. See that repo for a fix shortly!

### CODE REVIEW

File: run-all-tests.sh [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+# PERL_HASH_SEED=8 sh run-all-tests.sh -v t/plugin-yaml.t
 
 export PERL5LIB="$PWD/lib:$PERL5LIB";
 # export SWAGGER2_DEBUG=1;
 
 t () {
-  echo "\$ cd ../$PROJECT && prove -l $@";
-  cd ../$PROJECT && prove -l $@ || exit $?;
+  echo "\$ cd ../$PROJECT && prove -l $*";
+  cd ../$PROJECT && prove -l "$@" || exit $?;
```
Comment (jhthorsen) [inline run-all-tests.sh]: I don't get why you have quotes around `$@`. What does it fix? I would think that groups all arguments into a single argument when you pass it on to `prove`.

**Comment by mohawk2** [comment]
From the `bash` man page:

```
       @      Expands to the positional parameters, starting from  one.   When
              the  expansion  occurs  within  double  quotes,  each  parameter
              expands to a separate word.  That is, "$@" is equivalent to "$1"
              "$2"  ...   If the double-quoted expansion occurs within a word,
              the expansion of the first parameter is joined with  the  begin‐
              ning  part  of  the original word, and the expansion of the last
              parameter is joined with the last part  of  the  original  word.
              When  there  are no positional parameters, "$@" and $@ expand to
              nothing (i.e., they are removed).
```

File: run-all-tests.sh (hunk 1 above)
Comment (mohawk2) [reply run-all-tests.sh]: From the `bash` man page:

```
       @      Expands to the positional parameters, starting from  one.   When
              the  expansion  occurs  within  double  quotes,  each  parameter
              expands to a separate word.  That is, "$@" is equivalent to "$1"
              "$2"  ...   If the double-quoted expansion occurs within a word,
              the expansion of the first parameter is joined with  the  begin‐
              ning  part  of  the original word, and the expansion of the last
              parameter is joined with the last part  of  the  original  word.
              When  there  are no positional parameters, "$@" and $@ expand to
              nothing (i.e., they are removed).
```

Comment (mohawk2) [reply run-all-tests.sh]: Tl;dr:

```
# for a script run as `myscript "with    spaces" "hello"`
echo "$@" # preserves spaces
echo $@ # loses spaces, also has wrong number of args
```

**Comment by jhthorsen** [comment]
Cool beans 👍

**Pull request merged**